jetbrains-core/src/software/aws/toolkits/jetbrains/ui/connection/AwsConnectionSettingsSelector.kt

```diff
@@ -91,6 +90,7 @@ class AwsConnectionSettingsSelector(
         } catch (_: Exception) {
             view.credentialProvider.setSelectedInvalidCredentialsProvider(credentialProviderId)
         }
+        fireChange()
```

Comment:
How does this actually do anything? The listener is empty in LocalLambdaRunConfiguration?

Reply:
Ah I see, I thought this was a different bug